Tests/FluentAssertions.Equivalency.Specs/SelectionRulesSpecs.cs

```diff
@@ -506,53 +506,175 @@ public void When_a_property_is_protected_it_should_be_ignored()
         act.Should().NotThrow();
     }
 
-    [Fact]
-    public void When_a_property_is_hidden_in_a_derived_class_it_should_ignore_it()
+    public class MemberHiding
```

Comment:
@jnyrup can you please check the next five specs to see if they make sense to you too. Then I can mimic the same ones for the fields.

Reply:
They all make sense to me.